app/src/main/java/com/example/android/viewModel/RestroomListAdapter.java

```diff
@@ -30,21 +31,22 @@ public class RestroomListAdapter extends RecyclerView.Adapter<RestroomListAdapte
     private boolean _isEnabled = false;
 
     class RestroomViewHolder extends RecyclerView.ViewHolder implements View.OnClickListener, View.OnLongClickListener {
-        private final Store store = Store.getInstance();
-        private final TextView restroomListItemTitleView;
-        private final TextView restroomListItemContentView;
-        private final ImageView restroomListItemDeleteCheckBox;
-        private final View restroomListItemView;
-        final RestroomListAdapter mAdapter;
-
+        private final Store _store = Store.getInstance();
+        private final TextView _restroomListItemTitleView;
+        private final TextView _restroomListItemContentView;
+        private final ImageView _restroomListItemDeleteCheckBox;
+        private final View _restroomListItemView;
+        private RestroomsViewModel _viewModel;
```

Comment:
Suggest add `final` here.

```suggestion
        private final RestroomsViewModel _viewModel;
```